lib/widgets/drive/index.coffee

```diff
@@ -0,0 +1,119 @@
+###
+The MIT License
+
+Copyright (c) 2015 Resin.io, Inc. https://resin.io.
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+###
+
+_ = require('lodash')
+async = require('async')
+Promise = require('bluebird')
+drivelist = Promise.promisifyAll(require('drivelist'))
+form = require('resin-cli-form')
+InquirerList = require('inquirer/lib/prompts/list')
+UI = require('inquirer/lib/ui/baseUI')
+DriveScanner = require('./drive-scanner')
+
+_format = '<%= device %> (<%= size %>) - <%= description %>'
```

Comment:
Instead of encapsulating just the format, you can encapsulate the function that creates the interpolated string, which would allow you to remove `_.template(..)(..)` duplication in various places.